app/forms/collection_form.rb

```diff
@@ -41,7 +41,10 @@ def build_child
 
   def save
     if valid?
-      run_callbacks(:save) { collection.all?(&:save) }
+      run_callbacks(:save) do
```

Comment:
Nope it's not in a transaction. Do you think it should be. There are no validations on the model layer so it should never be false. I guess an exception might be raised in one of them if the DB died maybe? It's just a convenient way of returning true. Do you think we need to make it atomic?